redis/commands/helpers.py

```diff
@@ -64,6 +64,11 @@ def parse_list_to_dict(response):
     for i in range(0, len(response), 2):
         if isinstance(response[i], list):
             res["Child iterators"].append(parse_list_to_dict(response[i]))
+            try:
+                if isinstance(response[i + 1], list):
+                    res["Child iterators"].append(parse_list_to_dict(response[i + 1]))
```

Comment:
What happens with parse_list_to_dict if instead of res["Child iterators"] we pass in a res.get("Child iterators", []) (or None) as a default? Do we avoid the try/except cost?

Reply:
The try/except block is not for the res but for the response (response[i +1] can raise IndexError).